chapters/en/slides/module3/module3_05.md

```diff
@@ -0,0 +1,101 @@
+---
+type: slides
+title: 'module3_05'
+---
+
+# Pushing changes from a remote repository
+
+---
+# Communicating with the remote repository
```

Comment:
Isn't this slide more of  motivation to why we need to **pull** from GitHub? What do you think of putting it first in the next slide deck instead?